src/Groups/GroupDetails/GroupDetails.js

```diff
@@ -26,10 +26,14 @@ import {
   GROUP_ACCORDTION_LABELS,
 } from '../constants';
 import GroupInformation from './GroupInformation';
+import { LEDGER_ACCORDTION, LEDGER_ACCORDTION_LABELS } from '../../components/Ledger/constants';
```

Comment:
this import looks weird in current components structure. You create a new component, but use accordion id from another.

Reply:
Removed